plugins/dashboard_plugin.py

```diff
@@ -23,126 +23,112 @@
 class Dashboard(BaseView):
     template_folder = os.path.join(os.path.dirname(__file__), 'templates')
 
+    BILL_DAGS = (
+        'windowed_bill_scraping',
+        'daily_scraping',
+        'friday_hourly_scraping',
+        'saturday_hourly_scraping',
+    )
+
+    EVENT_DAGS = (
+        'windowed_event_scraping',
+        'daily_scraping',
+        'friday_hourly_scraping',
+        'saturday_hourly_scraping',
+    )
+
+    DATETIME_FORMAT = '%m/%d/%y %I:%M %p'
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.airflow_session = settings.Session()
```

Comment:
FYI, I changed these from class attributes to instance attributes because I don't think these are actually constants!